lib/connection.ex

```diff
@@ -71,7 +72,12 @@ defmodule Janus.Connection do
   defp do_start(method, transport_module, transport_args, handler_module, handler_args, options) do
     apply(GenServer, method, [
       __MODULE__,
-      {transport_module, transport_args, handler_module, handler_args},
+      %{
+        transport_module: transport_module,
```

Comment:
Personally I dont like this change but I could have not found solution to create a map from variable, where its key would be a variable name. Something like `%{transport_module, transport_args ...}`